cmd/statsdaemon/main.go

```diff
@@ -20,7 +20,7 @@ import (
 	"net/http"
 	_ "net/http/pprof"
 
-	"github.com/stvp/go-toml-config"
+	"github.com/rakyll/globalconf"
```

Comment:
Moved it to the grafana one. Also took out the vendor lock, assuming with it under grafana repo we want the latest always?